### PR TITLE
Update German translation (27 Oct 2017 UTC)

### PR DESCRIPTION
**Pull request by Wuzzy2**
Here's my update for the German translation of the client.

I've already ran the `copy_fix` update string thingie. Have fun! :-)

### CODE REVIEW

**Comment by Wuzzy2** [comment]
Some strings which confuse me atm:

- “Unfinished map”
- “Indicate map finish”

Someone in IRC said the translations were incorrect (wrong context).
Can someone please explain to me what these strings mean *exactly*?

File: data/languages/german.txt [hunk 1]
```diff
@@ -138,13 +138,13 @@ Delete demo
...
-== Aufnahmendetails:
+== Aufnahmedetails:
 
 Demofile: %s
-== Demodatei: %s
+== Aufnahmedatei: %s
 
 Demos
-== Aufnahm.
+== Aufnahmen
```
Comment (def-) [inline data/languages/german.txt]: Does it still fit into the space at 1280x1024 resolution?

Comment (Wuzzy2) [reply data/languages/german.txt]: Yes.

**Comment by savander** [comment]
The unfinished map is used on DDNet tab. If you check that option, it will filter maps which you finished and which not. Indicate map finish is the option, which can be used to enable a flag next to map (which tells you, that you already finished map).

File: data/languages/german.txt [hunk 2]
```diff
@@ -743,13 +735,13 @@ Look out!
 == Pass auf!
 
 Show names in chat in team colors
-== Nutze Teamfarben im Chat
+== Namen im Chat in Teamfarben anzeigen
 
 Enable team chat sound
-== Aktiviere Ton für Team-Nachrichten
+== Ton für Team-Nachrichten anzeigen
```
Comment (Jupeyy) [inline data/languages/german.txt]: aktivieren

File: data/languages/german.txt [hunk 3]
```diff
@@ -878,141 +870,184 @@ Save ghost
...
-== 
+== Netzwerk
 
 FPM
-== 
+== FPM
 
 Spree
-== 
+== Amoklauf
```
Comment (Jupeyy) [inline data/languages/german.txt]: I think Amoklauf isn't a good translation for killing spree, but more for rampage.
Spree is more meant as "Serie an Abschüssen".

File: data/languages/german.txt (hunk 2 above)
Comment (Wuzzy2) [reply data/languages/german.txt]: Done.

File: data/languages/german.txt (hunk 3 above)
Comment (Wuzzy2) [reply data/languages/german.txt]: Done.

**Comment by Wuzzy2** [comment]
> The unfinished map is used on DDNet tab. If you check  that option, it will filter maps which you finished and which not.  Indicate map finish is the option, which can be used to enable a flag  next to map (which tells you, that you already finished map).

Done

**Pull request merged**